### PR TITLE
[NETID] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**

### CODE REVIEW

File: dll/win32/netid/lang/ro-RO.rc [hunk 1]
```diff
@@ -1,7 +1,8 @@
 /*
- * FILE:       dll/win32/netid/lang/ro-RO.rc
- *             ReactOS Project (https://reactos.org)
- * TRANSLATOR: Ștefan Fulea (stefan dot fulea at mail dot com)
+ * FILE:          dll/win32/netid/lang/ro-RO.rc
+ *                ReactOS Project (https://reactos.org)
+ * TRANSLATORS:   Ștefan Fulea (stefan dot fulea at mail dot com)
+ *                Andrei Miloiu (miloiuandrei@gmail.com)
```
Comment (reactos-contributor) [inline dll/win32/netid/lang/ro-RO.rc:5]: You may also add the years of contribution here (say, the way you did it [here](https://github.com/reactos/reactos/blob/master/dll/cpl/input/lang/ro-RO.rc#L6)).

File: dll/win32/netid/lang/ro-RO.rc [hunk 2]
```diff
@@ -72,20 +73,20 @@ BEGIN
...
     1021 "Notă: Doar administratorii pot schimba identitatea calculatorului."
     1022 "Notă: Identitatea calculatorului nu poate fi schimbată deoarece:"
-    1029 "The new computer name ""%1"" is a number. The name may not be a number."
+    1029 "Noul nume de calculator ""%1"" e un număr. Numele nu poate fi un număr."
     1030 "Noul nume „%1” conține caractere nepermise. Caracterele nepermise sunt ` ~ ! @ # $ %% ^ & * ( ) = + [ ] { } \\ | ; : ' "" , < > / și ?"
-    1031 "The first domain of the domain name ""%1"" is a number. The first domain may not be a number."
-    1032 "The domain name ""%1"" contains characters which are not allowed. Standard DNS names may contain letters (a-z, A-Z), numbers (0-9), and hyphens, but no spaces. Periods (.) are used to separate domains. Example: domain-1.reactos.org."
+    1031 "Primul domeniu al numelui de domeniu ""%1"" este un număr. Primul domeniu nu poate fi un număr."
+    1032 "Numele de domeniu ""%1"" conține caractere ce nu sunt permise. Numele standard de DNS names pot conține litere (a-z, A-Z), numere (0-9), și cratime, dar nu spații. Punctele (.) sunt folosite la domeniile separate. De exemplu: domeniu-1.reactos.org."
```
Comment (reactos-contributor) [inline dll/win32/netid/lang/ro-RO.rc:89]: I think you forgot to remove "names" (after "DNS").

**Review by reactos-contributor — CHANGES_REQUESTED** [review]

File: dll/win32/netid/lang/ro-RO.rc (hunk 2 above)
Comment (AndreiMiloiu) [reply dll/win32/netid/lang/ro-RO.rc]: Yeah, I forgot it.

File: dll/win32/netid/lang/ro-RO.rc (hunk 1 above)
Comment (AndreiMiloiu) [reply dll/win32/netid/lang/ro-RO.rc]: I'm not sure I added the right years for you.

**Review by reactos-contributor — APPROVED** [review]

**Pull request merged**